cpp/src/components/connectivity.cu

```diff
@@ -78,7 +78,7 @@ std::enable_if_t<std::is_signed<VT>::value> connected_components_impl(
                                                    stream);
   } else {
     SCC_Data<ByteT, VT> sccd(nrows, graph.offsets, graph.indices);
-    sccd.run_scc(labels);
+    auto num_iters = sccd.run_scc(labels);
```

Comment:
Is this an unused variable, and if so should it be removed? I don't see it anywhere else in the diff.

Reply:
It's useful for debugging and harmless. If you build debug and put a breakpoint there, you immediately get the `num_iters`, as opposed to having to re-build to provide it. It's not for immediate consumption, but for debugging needs. Does not trigger a warning, so I'd leave it there.